src/app/home/work-item-widget/work-item-widget.component.ts

```diff
@@ -97,8 +97,13 @@ export class WorkItemWidgetComponent implements OnDestroy, OnInit  {
         active: true
       }])
       .switchMap(filters => this.workItemService
-        .getWorkItems(100000, filters))
+        .getWorkItems(100, filters)
+      )
       .map(val => val.workItems)
+      .map(workItems => workItems.filter(item => {
```

Comment:
this is pure duplicated code from above - any reason  why this bad temporary hack of filtering is not at least put into a shared method to avoid future deviation in the multiple places?

Reply:
No specific reason. I couldn't find any shared utils to put this method, but yeah I feel the problem and I'll create one.